src/test/scala/com/datastax/gatling/plugin/request/CqlRequestActionSpec.scala

```diff
@@ -88,7 +89,7 @@ class CqlRequestActionSpec extends BaseSpec with TestKitBase {
       capturedStatement shouldBe a[SimpleS]
       capturedStatement.getConsistencyLevel shouldBe null
       capturedStatement.getSerialConsistencyLevel shouldBe null
-      capturedStatement.getPageSize shouldBe 0
+      capturedStatement.getPageSize should be <= 0
```

Comment:
how's that possible that `getPageSize` will be `< 0`?

Reply:
I'm... not sure.  I'm gonna _guess_ this is a copy/paste error... ?

Reply:
Oh, that's right, this actually was correct.  If this was simply an =0 test we get the following failure:

```
[info] CqlRequestActionSpec:                                                    
[info] CQL                                                                      
[info] - should have default CQL attributes set if nothing passed *** FAILED ***
[info]   -2147483648 was not equal to 0 (CqlRequestActionSpec.scala:91)  
```

That actually matches up to the Javadoc:

```
Returns:
the set page size, otherwise 0 or a negative value to use the default value defined in the configuration.
```